src/evict/evict_lru.c

```diff
@@ -1012,7 +1019,7 @@ retry:	while (slot < max_entries && ret == 0) {
 				cache->evict_file_next = NULL;
 			else
 				dhandle = TAILQ_FIRST(&conn->dhqh);
-		} else {
+		} else if (!F_ISSET(dhandle, WT_DHANDLE_DEAD) || dead_next) {
```

Comment:
> We use the __wt_page_evict_soon to try to get a page evicted before it causes problems - I'm concerned focusing eviction too heavily on a single file will limit how well that works.

@agorrod, I agree, that's a bigger change than I want to make and I've reverted this change.